README.md

```diff
@@ -0,0 +1,5 @@
+Happy Birthday! :robot: 
+
+I hope you're having a great day.
```

Comment:
The `.` at the end sounds passive aggressive.

Reply:
That's the tone I was aiming for.
.